README.md

```diff
@@ -444,7 +444,7 @@ Complete installation requirements are the same as for Strapi itself and can be
 
 **Supported Meilisearch versions**:
 
-This package only guarantees the compatibility with the [version v0.26.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.26.0).
+This package only guarantees the compatibility with the [version v0.29.1 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.26.0).
```

Comment:
You use [this version of Meilisearch JS](https://github.com/meilisearch/meilisearch-js/releases/tag/v0.28.0) which is only compatible with Meilisearch v0.29.0, not v0.29.1. Did I miss something here?

Reply:
The `1` in `0.29.1` is the patch it doesn't introduce breaking changes, so every version of v.0.29.* should still be compatible

Reply:
```suggestion
This package only guarantees the compatibility with the [version v0.29.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.29.0).
```